--- a/drivers/net/phy/Kconfig
+++ b/drivers/net/phy/Kconfig
@@ -355,6 +355,17 @@ config AIROHA_EN8801SC_PHY
         ---help---
           Currently supports the Airoha EN8801S PHY for MT7981 SoC.
 
+config AIROHA_EN8811H_PHY
+	tristate "Drivers for Airoha EN8811H 2.5G Gigabit PHY"
+	---help---
+	  Currently supports the Airoha EN8811H PHY.
+
+config AIROHA_EN8811H_PHY_DEBUGFS
+	bool "EN8811H debugfs support"
+	depends on AIROHA_EN8811H_PHY
+	---help---
+	  Enable creation of debugfs files for the EN8811H drivers.
+
 config ADIN_PHY
 	tristate "Analog Devices Industrial Ethernet PHYs"
 	help
--- a/drivers/net/phy/Makefile
+++ b/drivers/net/phy/Makefile
@@ -69,6 +69,8 @@ aquantia-objs			+= aquantia_hwmon.o
 endif
 obj-$(CONFIG_AIROHA_EN8801S_PHY)	+= en8801s.o
 obj-$(CONFIG_AIROHA_EN8801SC_PHY)	+= en8801sc.o
+air_en8811h-y := air_en8811h_main.o air_en8811h_api.o
+obj-$(CONFIG_AIROHA_EN8811H_PHY)	+= air_en8811h.o
 obj-$(CONFIG_AQUANTIA_PHY)	+= aquantia.o
 obj-$(CONFIG_AX88796B_PHY)	+= ax88796b.o
 obj-$(CONFIG_AT803X_PHY)	+= at803x.o
